src/test/resources/specs/name-matching/Full_stops_in_names.feature

```diff
@@ -70,7 +70,7 @@ Feature: Names with full stops
     Then the following identities will be tried in this order
       | First name  | Last name | Date of Birth | nino      |
       | Aaa Bb. Ccc | Ddd       | 1987-12-10    | SE123456B |
-      | Bb. Ccc     | Ddd       | 1987-12-10    | SE123456B |
+      | Bb.         | Ddd       | 1987-12-10    | SE123456B |
```

Comment:
I worry that this is a behaviour change - I guess the order doesn't matter but we must make sure we still have a test to prove that we use "St. John" etc. as a single word name in name matching.

Perhaps the issue is that Bb. is three characters. Changing the test so that the candidate's first name is B. Ccc might be the solution?